libgadget/forcetree.c

```diff
@@ -126,11 +153,11 @@ ForceTree force_tree_build(int npart, DomainDecomp * ddecomp)
         {
             force_tree_free(&tree);
 
-            message(0, "TreeAllocFactor from %g to %g\n", All.TreeAllocFactor, All.TreeAllocFactor*1.15);
+            message(0, "TreeAllocFactor from %g to %g\n", ForceTreeParams.TreeAllocFactor, ForceTreeParams.TreeAllocFactor*1.15);
 
-            All.TreeAllocFactor *= 1.15;
+            ForceTreeParams.TreeAllocFactor *= 1.15;
```

Comment:
This param is mutable. We may want to add a comment to indicate this -- usually 'param's are immutable.